include/nsysccr/cdc.h

```diff
@@ -21,9 +21,10 @@ typedef struct CCRCDCWowlWakeDrcArg CCRCDCWowlWakeDrcArg;
 typedef struct CCRCDCUicConfig CCRCDCUicConfig;
 typedef struct CCRCDCFWInfo CCRCDCFWInfo;
 typedef struct CCRCDCSoftwareVersion CCRCDCSoftwareVersion;
+typedef struct CCRCDCSysDrhState CCRCDCSysDrhState;
```

Comment:
Thoughts about naming this `CCRCDCDrhState` and `CCRCDCDrhStateEnum` to be consistent with the other typedefs?

Reply:
done